### PR TITLE
Memento pattern design

### PR DESCRIPTION
**Pull request by Alex-CG**

### CODE REVIEW

File: src/main/java/edu/learning/patterns/behavioral/memento/MementoPattern.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        System.out.println(notepad.description);
+
+        careTaker.restore(notepad);
+        System.out.println(notepad.description);
+
+    }
+
+}
+class Notepad {
+
```
Comment (miguelReyesC) [inline src/main/java/edu/learning/patterns/behavioral/memento/MementoPattern.java]: Add getters and setters

File: src/main/java/edu/learning/patterns/behavioral/memento/MementoPattern.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        Memento(String name, String description){
+            this.name = name;
+            this.description = new StringBuilder(description);
+        }
+    }
+
+}
+
+
+class CareTaker {
```
Comment (miguelReyesC) [inline src/main/java/edu/learning/patterns/behavioral/memento/MementoPattern.java]: Add getters and setters

**Review by miguelReyesC-zz — CHANGES_REQUESTED** [review]
Please improve

**Comment by miguelReyesC** [comment]
Instale un plugin para que haga code review automatico, fixea lo que te dice :P

**Review by miguelReyesC-zz — APPROVED** [review]

**Pull request merged**